src/api/errors/OperatorError.ts

```diff
@@ -0,0 +1,140 @@
+import log from 'loglevel'
```

Comment:
Can we move the core API for CoW protocol to `src/api/cow` instead of just being in `src/api`. We expect to have more APIs. 

Actually, you already included the Gnosis Safe API estimation of gas prices (although i think we should probably remove that).